.github/workflows/docker.yaml

```diff
@@ -26,8 +29,12 @@ jobs:
       - name: Install Dev Requirements
         run: pip install -r dev-requirements.txt
 
-      - name: Build Fidesctl
+      - name: Build Fidesctl Prod
         run: nox -s "build(prod)"
 
-      - name: Push Fidesctl
-        run: nox -s push
+      - name: Push Fidesctl Dev
+        run: nox -s "push(dev)"
```

Comment:
Does there need to be build step for the dev image?

Reply:
Yea, I would expect there to be a build step for dev. Also, you can separate buil/push dev and build/push prod into parallel jobs that depend on the login job. That would speed things up on workflow runs when both dev and prod are needed, but it wouldn't allow you to bail out of a prod build/push job if dev fails. Food for thought.

Reply:
no, `build(prod)` is used regardless of what tag we're using when we push to Dockerhub